README.md

```diff
@@ -1 +1,38 @@
 # Haskell Errors
+
+So you would like to help improve the error messages we see in the Haskell toolchain and ecosystem?
+
+Welcome! We're glad to have you, and we will do our best to ensure your efforts are worthwhile.
+
+
```

Comment:
```suggestion
This repository is unrelated to Haskell's [`errors` package](https://hackage.haskell.org/package/errors), which [lives elsewhere](https://github.com/Gabriel439/Haskell-Errors-Library).
```

The Haskell Infrastructure team requested that we publicly delineate ourselves from `errors`.